src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1391,6 +1391,15 @@
   <data name="DoNotNameEnumValuesReservedTitle" xml:space="preserve">
     <value>Do not name enum values 'Reserved'</value>
   </data>
+  <data name="DoNotOverloadOperatorEqualsOnReferenceTypesDescription" xml:space="preserve">
+    <value>For reference types, the default implementation of the equality operator is almost always correct. By default, two references are equal only if they point to the same object.</value>
+  </data>
+  <data name="DoNotOverloadOperatorEqualsOnReferenceTypesMessage" xml:space="preserve">
+    <value>'{0}' should not overload the equality operator.</value>
+  </data>
+  <data name="DoNotOverloadOperatorEqualsOnReferenceTypesTitle" xml:space="preserve">
+    <value>Do not overload operator '==' on reference types</value>
```

Comment:
A quick search shows that the operator is named `=` in VB, and `==` in C#; so this message isn't language-agnostic.

Reply:
I might be wrong but from what I have seen the messages do not seem to be language agnostic. @mavasani What's the "rule"?

Reply:
We should try to keep the messages language agnostic - our existing messages which were ported from original FxCop implementation may violate this at some places, but that you should not encourage us to add more of these. @Evangelink - can you file a tracking issue for all messages that you came across that were not language agnostic?